deepchem/models/torch_models/layers.py

```diff
@@ -3132,12 +3132,27 @@ def __init__(self,
         self.name: str = name
         self.nodes: int = nodes
 
-        self.dense1: nn.ModuleList = nn.ModuleList(
-            [nn.Linear(nodes, self.units) for _ in range(edges - 1)])
+        # Case when >2 inputs are passed
+        if "prev_shape":
```

Comment:
This is a type. I think you mean 
```
if prev_shape:
```
With a string, this branch will always activate

Reply:
yup, was a typo, fixed it